backend/src/main/java/com/pickpick/slackevent/application/SlackEvent.java

```diff
@@ -12,7 +12,8 @@ public enum SlackEvent {
     MESSAGE_CHANGED("message", "message_changed"),
     MESSAGE_DELETED("message", "message_deleted"),
     CHANNEL_RENAME("channel_rename", ""),
-    CHANNEL_DELETED("channel_deleted", "");
+    CHANNEL_DELETED("channel_deleted", ""),
+    MEMBER_CHANGED("user_profile_changed", "");
```

Comment:
```suggestion
    CHANNEL_DELETED("channel_deleted", ""),
    MEMBER_CHANGED("user_profile_changed", ""),
    ;
```
앞으로 enum 관리할 때 이렇게 하면 어떨까요? 매번 마지막줄이 변경되서 여러 작업이 함께 일어날 경우 이부분에서 충돌이 일어날 수 있을 것 같아요!
물론 쉽게 해결할 수 있는 충돌이지만, 그래도 방지하면 좋을 것 같아 리뷰 남깁니다 :)

Reply:
미션에서 제공됐던 enum이 봄 제안처럼 되어있어서
처음엔 어색하다 느꼈는데요, 개발을 진행하며 이유가 느껴지더라고요
저도 Enum선언 이후 세미콜론을 개행해서 추가하는 게 나은 것 같습니다
사소하긴 하지만요 :)

Reply:
왜 마지막에 굳이 쉼표를 찍어둘까? 했었는데 이런 이유였군요  봄 덕분에 알았어요!! 😁